.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+ - repo: local
+   hooks:
+   -   id: pylint
+       name: pylint
+       entry: pylint
+       language: system
+       types: [python]
+ - repo: local
+   hooks:
+   -   id: clang-format
+       name: clang-format
+       entry: clang-format -i
```

Comment:
With "clang-format-10` that is available in this Ubuntu 20.04 upgrade we could use `clang-format-10 --dry-run --Werror` if we don't like the automatic pre-commit formatting.